zipkin/src/main/java/com/linecorp/armeria/common/tracing/RequestContextCurrentTraceContext.java

```diff
@@ -50,9 +50,20 @@
      * Use this singleton when building a {@link brave.Tracing} instance for use with
      * {@link HttpTracingService} or {@link HttpTracingClient}.
      *
+     * <p>If you need to customize the context, use {@linkplain #newBuilder()} instead.
+     *
      * @see brave.Tracing.Builder#currentTraceContext(brave.propagation.CurrentTraceContext)
      */
-    public static final CurrentTraceContext INSTANCE = new RequestContextCurrentTraceContext();
+    public static final CurrentTraceContext DEFAULT = new RequestContextCurrentTraceContext(new Builder());
+
+    /**
+     * Singleton retained for backwards compatibility, but replaced by {@link #DEFAULT}.
+     *
+     * @deprecated Please use {@link #DEFAULT} or {@link #newBuilder()} to customize an instance.
+     */
+    @Deprecated
+    public static final CurrentTraceContext INSTANCE = new RequestContextCurrentTraceContext(new Builder());
```

Comment:
Could be just `= DEFAULT`?

Reply:
addressed in the other PR (will rebase this once merged)